charts/pg-db/values.yaml

```diff
@@ -1,134 +1,289 @@
+#        key: role
+#        operator: Equal
+#        value: connection-poolers
+#
+#    global:
+#      repo1-retention-full: "14"
+#      repo1-retention-full-type: time
+#      repo1-path: /pgbackrest/postgres-operator/cluster1/repo1
+#      repo1-cipher-type: aes-256-cbc
+#      repo1-s3-uri-style: path
+#      repo2-path: /pgbackrest/postgres-operator/cluster1-multi-repo/repo2
+#      repo3-path: /pgbackrest/postgres-operator/cluster1-multi-repo/repo3
+#      repo4-path: /pgbackrest/postgres-operator/cluster1-multi-repo/repo4
+#    repoHost:
+#      priorityClassName: high-priority
+#
+#      topologySpreadConstraints:
+#      - maxSkew: 1
+#        topologyKey: my-node-label
+#        whenUnsatisfiable: ScheduleAnyway
+#        labelSelector:
+#          matchLabels:
+#            postgres-operator.crunchydata.com/pgbackrest: ""
+#      affinity:
+#        podAntiAffinity:
+#          preferredDuringSchedulingIgnoredDuringExecution:
+#           - weight: 1
+#           podAffinityTerm:
+#             labelSelector:
+#             matchLabels:
+#               postgres-operator.crunchydata.com/cluster: keycloakdb
+#               postgres-operator.crunchydata.com/role: pgbouncer
+#             topologyKey: kubernetes.io/hostname
+  manual:
+   repoName: repo1
+   options:
+   - --type=full
+  repos:
+  - name: repo1
+    schedules:
+     full: "0 0 * * 6"
+#    differential: "0 1 * * 1-6"
+     volume:
+      volumeClaimSpec:
+       accessModes:
+       - ReadWriteOnce
+       resources:
+        requests:
+         storage: 1Gi
+#    - name: repo2
+#      s3:
+#        bucket: "<YOUR_AWS_S3_BUCKET_NAME>"
+#        endpoint: "<YOUR_AWS_S3_ENDPOINT>"
+#        region: "<YOUR_AWS_S3_REGION>"
+#    - name: repo3
+#      gcs:
+#        bucket: "<YOUR_GCS_BUCKET_NAME>"
+#    - name: repo4
+#      azure:
+#        container: "<YOUR_AZURE_CONTAINER>"
+#
+#  restore:
+#    enabled: true
+#    repoName: repo1
+#    options:
+#     PITR restore in place
+#     - --type=time
+#     - --target="2021-06-09 14:15:11-04"
+#     restore individual databases
+#     - --db-include=hippo
 
-replicas:
-  size: 2
-  resources:
-    requests:
-      cpu: "1"
-      memory: "128Mi"
-    limits:
-      cpu: "1"
-      memory: "128Mi"
-  volumeSpec:
-    accessmode: ReadWriteOnce
-    size: 1G
-    storagetype: dynamic
-    storageclass: "standard"
-  enableSyncStandby: false
-  expose:
-    serviceType: ClusterIP
+pmm:
+ enabled: false
+ image:
+  repository: percona/pmm-client
+  tag: 2.37.0
+#   imagePullPolicy: IfNotPresent
+ secret: cluster1-pmm-secret
+ serverHost: monitoring-service
+#   resources:
+#     requests:
+#       memory: 200M
+#       cpu: 500m
 
-pgBadger:
-  enabled: false
-  image: ""
-  port: 10000
+# patroni:
+#   dynamicConfiguration:
+#     postgresql:
+#       parameters:
+#         max_parallel_workers: 2
+#         max_worker_processes: 2
+#         shared_buffers: 1GB
+#         work_mem: 2MB
 
 secrets:
-  name:
+ name:
```

Comment:
bad indentation?

Reply:
Lint check passed. Any ideas how to check it? 
I checked with 
`ct lint --config .github/ct.yaml --lint-conf .github/lintconf.yaml`
it was successful, with 2 spaces check failed.

Reply:
I think it's confused because of inconsistency, when you specify 2 spaces everywhere the lint is ok. I just used some online yaml formatter and it formatted the whole yaml with 2 spaces and it passes the lint.